po/extra/pt.po

```diff
@@ -0,0 +1,359 @@
+# Portuguese translations for extra package.
+# Copyright (C) 2020 THE extra'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# Automatically generated, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-12-04 13:13+0900\n"
+"PO-Revision-Date: 2021-01-04 11:59+0000\n"
+"Last-Translator: Hugo Carvalho <hugokarvalho@hotmail.com>\n"
+"Language-Team: Portuguese (pt)\n"
+"Language: pt\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.4.1\n"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:7
+#: data/com.github.donadigo.eddy.desktop.in:4
+msgid "Eddy"
+msgstr "Eddy"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:8
+msgid "Install and Manage .deb Files"
+msgstr "Instalar e gerir ficheiros .deb"
+
+#: data/com.github.donadigo.eddy.appdata.xml.in:10
+msgid "Install, uninstall and view information about debian packages."
+msgstr "Instalar, desinstalar e ver informação sobre pacotes debian."
```

Comment:
Capitalization, as suggested in [another comment](https://github.com/donadigo/eddy/pull/96/files#r551283476):

```suggestion
msgstr "Instalar, desinstalar e ver informação sobre pacotes Debian."
```

Reply:
Fixed